### PR TITLE
Added 360 degree F5 Mod

### PR DESCRIPTION
**Pull request by CoalCoding**
Default Keybind - P

### CODE REVIEW

File: src/main/java/cc/hyperium/mods/perspective/PerspectiveModifierContainer.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+public class PerspectiveModifierContainer {
+    public boolean enabled = false;
+
+    public float modifiedYaw;
+    public float modifiedPitch;
+
+    public void onEnable() {
+        if (!enabled) {
+            GeneralChatHandler.instance().sendMessage("Enabled F5 Perspective.");
```
Comment (supertassu) [inline src/main/java/cc/hyperium/mods/perspective/PerspectiveModifierContainer.java]: sending message every time perspective is toggled will be spammy and annoying, at least add option to disable it pls

also what does this have to do with f5

File: src/main/java/cc/hyperium/mods/perspective/PerspectiveModifierContainer.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            modifiedPitch = Minecraft.getMinecraft().thePlayer.cameraPitch;
+        } else {
+            onDisable();
+        }
+    }
+
+    public void onDisable() {
+        if (enabled) {
+            Minecraft.getMinecraft().gameSettings.thirdPersonView = 0;
+            GeneralChatHandler.instance().sendMessage("Disabled F5 Perspective.");
```
Comment (supertassu) [inline src/main/java/cc/hyperium/mods/perspective/PerspectiveModifierContainer.java]: spammy

Comment (CoalCoding) [reply src/main/java/cc/hyperium/mods/perspective/PerspectiveModifierContainer.java]: F5 toggles perspective in vanilla Minecraft, as does this, it's an improvement on it. Also toggling the output will need to be applied to several other areas aswell.

File: src/main/java/cc/hyperium/mods/perspective/PerspectiveModifierContainer.java (hunk 1 above)
Comment (CoalCoding) [reply src/main/java/cc/hyperium/mods/perspective/PerspectiveModifierContainer.java]: See reply to your other comment.
